### PR TITLE
Issue 4031: Updated cluster_install.md page for Ubuntu 22.04 and simp…

### LINKED ISSUE #4031: Documentation: Update agent cluster install docs
### What is the current documentation state?

https://github.com/open-horizon/open-horizon.github.io/blob/master/docs/anax/docs/cluster_install.md

Updating to ensure it works with Ubuntu 22.04.4 LTS on both arm64 and amd64.

### Where is this stated?

_No response_

### Why do you want to improve the statement?

_No response_

### Proposed Statement

_No response_

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by joewxboy**
…lified

# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #4031 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Ran locally in VM

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings
- [X] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [X] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

This closes issue #4031

### CODE REVIEW

File: docs/cluster_install.md [hunk 1]
```diff
@@ -374,6 +399,30 @@ This content describes how to install the Open Horizon agent on k3s or microk8s
...
+
+    c. Install a cluster-scoped agent using the local configuration:
+
+    ```bash
+    ./agent-install.sh -D cluster -i anax: -c css: -k ./agent-install.cfg --namespace $AGENT_NAMESPACE
+    ```
+
 7. Verify that the agent pod is running:
 
     ```bash
```
Comment (dlarson04) [inline docs/cluster_install.md:428]: not part of your PR but the examples above show setting the AGENT_NAMESPACE env variable.... but here we tell them if not set to use `openhorizon-agent`...  I think we should stick with the $AGENT_NAMESPACE syntax to be consistent throughout

**Review by dlarson04 — CHANGES_REQUESTED** [review]

File: docs/cluster_install.md [hunk 2]
```diff
@@ -374,6 +399,30 @@ This content describes how to install the Open Horizon agent on k3s or microk8s
     ./agent-install.sh -D cluster -i anax: -c css: -k css: --namespace $AGENT_NAMESPACE --namespace-scoped
```
[the remark this answers is not in the document]
Comment (joewxboy) [reply docs/cluster_install.md:399]: Step One at the top of the document has the user `sudo`.  We are assuming that this step is in the same terminal session.  Would it be helpful to have a reminder at this step?

**Comment by joewxboy** [comment]
@dlarson04 I made most of your suggested changes.  Two I had replies to that you may want to review.

**Review by euconnor — APPROVED** [review]

File: docs/cluster_install.md [hunk 3]
```diff
@@ -374,6 +399,30 @@ This content describes how to install the Open Horizon agent on k3s or microk8s
...
     ```
 
+    **Note** If your server is not configured with the certificate, config, and agent files you will need to use the following alternative steps to install:
```
[the remark this answers is not in the document]
Comment (dlarson04) [reply docs/cluster_install.md:402]: Yes... it always does

https://github.com/open-horizon/devops/blob/master/mgmt-hub/deploy-mgmt-hub.sh#L1385

**Review by dlarson04 — CHANGES_REQUESTED** [review]

File: docs/cluster_install.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (joewxboy) [reply docs/cluster_install.md:399]: Adding a reminder in this section.

File: docs/cluster_install.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (joewxboy) [reply docs/cluster_install.md:402]: OK, removed that set of alternate instructions.

Added the missing Note colons. :)

**Comment by joewxboy** [comment]
OK, I think I've made all the changes now @dlarson04 and @euconnor

**Review by dlarson04 — APPROVED** [review]

**Pull request merged**